src/main/java/mods/eln/Eln.java

```diff
@@ -5279,10 +5279,10 @@ private void recipeElectricalCable() {
 
         addRecipe(lowVoltageLQCableDescriptor.newItemStack(12), //Shoddy Low Voltage Cable
             "WWW",
-            "GCG",
+            "DCD",
             "WWW",
             'C', "ingotCopper",
-            'G', new ItemStack(Blocks.gravel),
+            'D', new ItemStack(Blocks.dirt),
```

Comment:
That would just crumble. Granted, so would the dirt. How about clay?

Reply:
I'm thinking of just getting rid of the 'filler' item so that it only costs 1 ingot and some wood